data/pilots/first-order/tie-se-bomber.json

```diff
@@ -219,6 +219,37 @@
       "image": "https://infinitearenas.com/xw2/images/pilots/sienarjaemustestpilot.png",
       "keywords": ["TIE"],
       "epic": true
+    },
+    {
+      "name": "Jul Jerjerrod",
```

Comment:
Need charges on Jul, right? Or maybe they're here and I'm just not seeing them?

Reply:
fixed!